services/api/tests/src/test_admin_org.py

```diff
@@ -282,20 +286,52 @@ def test_delete_org(mock_query_db, mock_write_db):
     mock_write_db.assert_has_calls(calls)
     assert actual_result == expected_result
 
+
 @patch("api.src.admin_org.pgquery.query_db")
 def test_delete_org_failure_orphan_rsu(mock_query_db):
```

Comment:
While the API unit tests pass, we seem to be missing tests for all of the new python scripts you've added. We will want to  add some tests for these.

Reply:
Added and passing